lib/attrsets.nix

```diff
@@ -302,6 +302,35 @@ rec {
         in mapAttrs g set;
     in recurse [] set;
 
+  /* Builds a map from <attr>=value to <attr>.<item>=value for each item in
+   * the list.
+   *
+   * This is mainly useful for flake-like structures.
+   *
+   * Type:
+   *   mapAttrsJump ::
+   *     [String] -> (String -> AttrSet) -> AttrSet<String, AttrSet>
+   *
+   * Example:
+   *   mapAttrsJump ["x86_64-linux"] (system: { hello = 42; })
+   *   => { x86_64-linux = { hello = 42; }; }
```

Comment:
This example looks like the regular `genAttrs`. Probably should be something like `{ hello = { x86_64-linux = 42; }; }`.